pkg/pool-hooks/test/foundry/StableSurgeHookUnit.t.sol

```diff
@@ -18,19 +18,20 @@ import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/Fixe
 import { ScalingHelpers } from "@balancer-labs/v3-solidity-utils/contracts/helpers/ScalingHelpers.sol";
 import { StablePool } from "@balancer-labs/v3-pool-stable/contracts/StablePool.sol";
```

Comment:
Less important for test files, but my OCD notes interface and contract imports aren't separated here.